src/screens/Settings/index.tsx

```diff
@@ -207,9 +211,11 @@ function Settings() {
         const {
           cloud_save_enabled: cloudSaveEnabled,
           save_folder: saveFolder,
-          title: gameTitle
+          title: gameTitle,
+          is_mac_native
         } = await getGameInfo(appName)
         setTitle(gameTitle)
+        setIsNativeMac(is_mac_native)
```

Comment:
Here is a good example for the variable spelling. Is this on porpuse ?

Reply:
`is_mac_native` is a field from the library so belongs to the GameInfo. So to follow the standard we have today I decided to keep like this.

Reply:
I would love to have setIsMacNative. So it follows the spelling of is_mac_native.